tpr/tpr.go

```diff
@@ -26,17 +26,15 @@ type Config struct {
 
 	// Settings.
 
-	// Name of the kind of ThirdPartyObjects. It should be in lower case
-	// and hyphen delimited. Kind names will be converted to CamelCase
-	// when creating ThirdPartyObjects. Hyphens in the kind are assumed to
-	// be word breaks. For instance the kind camel-case would be converted
-	// to CamelCase but camelcase would be converted to Camelcase.
+	// Name takes the form <kind>.<domain>. The domain is also known as
```

Comment:
https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource/

^ it's so confusing here, I don't know which one to use to make it less confusing. Most of the comment is copied from the URL above.

Then in the same doc you see:

```
$ kubectl get crontab
NAME                 KIND
my-new-cron-object   CronTab.v1.stable.example.com
```

Then you think, what is name, kind, group, and domain?

It's obvious that group==domain. I don't know which one to use. I also think that when it's domain `cluster.giantswarm.io` doesn't make sense. 

You see, it's complicated :)

- - -

But ok, let's stick with group, bc I think this is what we use most of the time.
My proposition is to change godoc to 

```
Name takes the form <kind>.<group>. The group is also known as a domain. [...]
```

And no more `domain` word in this file.

WDYT?